packages/hethers/package.json

```diff
@@ -1,34 +1,34 @@
 {
   "browser-old": "./dist/ethers.umd.js",
   "dependencies": {
-    "@ethersproject/abi": "5.5.0",
-    "@ethersproject/abstract-provider": "5.5.1",
-    "@ethersproject/abstract-signer": "5.5.0",
-    "@ethersproject/address": "5.5.0",
-    "@ethersproject/base64": "5.5.0",
-    "@ethersproject/basex": "5.5.0",
+    "@hethers/abstract-provider": "1.0.0",
+    "@hethers/abstract-signer": "1.0.0",
+    "@hethers/address": "1.0.0",
     "@ethersproject/bignumber": "5.5.0",
     "@ethersproject/bytes": "5.5.0",
-    "@ethersproject/constants": "5.5.0",
-    "@ethersproject/contracts": "5.5.0",
+    "@hethers/constants": "1.0.0",
+    "@hethers/contracts": "1.0.0",
     "@ethersproject/hash": "5.5.0",
-    "@ethersproject/hdnode": "5.5.0",
-    "@ethersproject/json-wallets": "5.5.0",
+    "@hethers/hdnode": "1.0.0",
+    "@hethers/json-wallets": "1.0.0",
+    "@hethers/logger": "1.0.0",
+    "@hethers/networks": "1.0.0",
+    "@hethers/providers": "1.0.0",
+    "@ethersproject/rlp": "5.5.0",
```

Comment:
Maybe we should remove the `rlp` package since we do not need it

Reply:
`rlp` package is removed